Failing lustre tests

Running pytest on branch v090dev with lustre mounted fails on these tests:

```
FAILED pyaerocom/io/test/test_read_aeronet_sdav3.py::test_load_berlin_AeroSdaV2L2D - OSError: No files could be detected that match the pattern
FAILED pyaerocom/io/test/test_read_aeronet_sunv3.py::test_load_berlin - OSError: No files could be detected that match the pattern
FAILED pyaerocom/io/test/test_read_ghost.py::TestReadGhost::test__eval_flags_slice - PermissionError: [Errno 13] Permission denied: b'/lustre/storeA/project/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EEA_AQ_...
FAILED pyaerocom/io/test/test_read_ghost.py::TestReadGhost::test_read_file[ghost_eea_daily-2290-Bleak House] - PermissionError: [Errno 13] Permission denied: b'/lustre/storeA/project/aerocom/aerocom1/AEROCOM_...
FAILED pyaerocom/io/test/test_readgridded.py::test_years_available - AssertionError: 
FAILED pyaerocom/io/test/test_readungridded.py::test_basic_attributes - AssertionError: assert ['od340aer', ...4&87aer', ...] == ['od340aer', ...er_calc', ...]
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_ungriddeddata_jungfraujoch - assert (17520, 12) == (245448, 12)
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch - assert 8323 == 25789
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch_lev3 - assert 8323 == 25789
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch_subset - pyaerocom.exceptions.DataCoverageError: ['sc550aer'] data could not be retrieved for meta  index (or station name) [0.0, 1.0]
FAILED pyaerocom/test/test_obsdata_versions.py::test_revision_dates - AssertionError: 
```

The two tests related to read_ghost is caused by the new ghost data having more restrictive permissions than other aerocom data. @hansbrenna can you update that? Other files have 664 permissions so they are readable but not writable even if not in the aerocom group.

For the others I'm not sure what is going on. Has the datasets been changed recently?

--- a/pyaerocom/conftest.py
+++ b/pyaerocom/conftest.py
@@ -145,6 +145,11 @@ def aeronet_sun_subset_reader():
     return reader
 
 @pytest.fixture(scope='session')
+def aeronet_sda_subset_reader():
+    reader = ReadAeronetSdaV3('AeronetSDAV3L2Subset.daily')
+    return reader
+
+@pytest.fixture(scope='session')
 def aeronetsunv3lev2_subset(aeronet_sun_subset_reader):
     r = aeronet_sun_subset_reader
     #return r.read(vars_to_retrieve=TEST_VARS)

--- a/pyaerocom/conftest.py
+++ b/pyaerocom/conftest.py
@@ -156,6 +156,12 @@ def data_scat_jungfraujoch():
     return r.read('sc550aer', station_names='Jungfrau*')
 
 @pytest.fixture(scope='session')
+def data_scat_jungfraujoch_full():
+    r = ReadEbas()
+    return r.read('sc550aer', station_names='Jungfrau*')
+
+
+@pytest.fixture(scope='session')
 def loaded_nasa_ames_example():
     from pyaerocom.io.ebas_nasa_ames import EbasNasaAmesFile
     #fp = TESTDATADIR.joinpath(TEST_PATHS['nasa_ames_sc550aer'])

--- a/pyaerocom/io/test/test_read_aeronet_sdav3.py
+++ b/pyaerocom/io/test/test_read_aeronet_sdav3.py
@@ -10,12 +10,11 @@ import os
 from pyaerocom.conftest import TEST_RTOL, lustre_unavail
 from pyaerocom.io.read_aeronet_sdav3 import ReadAeronetSdaV3
 
-@lustre_unavail
-def test_load_berlin_AeroSdaV2L2D():
-    reader = ReadAeronetSdaV3()
-    files = reader.find_in_file_list('*Berlin*')
+def test_load_thessaloniki(aeronet_sda_subset_reader):
+    reader = aeronet_sda_subset_reader
+    files = reader.find_in_file_list('*Thessaloniki*')
     assert len(files) == 1
-    assert os.path.basename(files[0]) == 'Berlin_FUB.lev30', files[0]
+    assert os.path.basename(files[0]) == 'Thessaloniki.lev30', files[0]
 
     test_vars = ['ang4487aer',
                  'od550aer',
@@ -30,15 +29,14 @@ def test_load_berlin_AeroSdaV2L2D():
     assert all([len(data[x]) > 100 for x in test_vars])
 
     assert isinstance(data['dtime'][0], np.datetime64)
-    assert data['dtime'][0] == np.datetime64('2014-07-06T12:00:00'), data['dtime'][0]
+    assert data['dtime'][0] == np.datetime64('2003-06-01T12:00:00'), data['dtime'][0]
 
     means = []
     for var in test_vars:
+        means.append(np.nanmean(data[var]))
 
-        means.append(data[var].mean())
-
-    desired = [1.3437323123209168, 0.15558341331479467, 0.02798509404146714,
-               0.12759832110794386]
+    desired = [1.4777584841303428, 0.1988665578854858, 0.036805761707404114,
+               0.16206080598741934]
 
     npt.assert_allclose(actual=means,
                         desired = desired,

--- a/pyaerocom/io/test/test_read_aeronet_sdav3.py
+++ b/pyaerocom/io/test/test_read_aeronet_sdav3.py
@@ -7,9 +7,10 @@ import pytest
 import numpy.testing as npt
 import numpy as np
 import os
-from pyaerocom.conftest import TEST_RTOL, lustre_unavail
+from pyaerocom.conftest import TEST_RTOL, testdata_unavail
 from pyaerocom.io.read_aeronet_sdav3 import ReadAeronetSdaV3
 
+@testdata_unavail
 def test_load_thessaloniki(aeronet_sda_subset_reader):
     reader = aeronet_sda_subset_reader
     files = reader.find_in_file_list('*Thessaloniki*')

--- a/pyaerocom/io/test/test_read_aeronet_sunv3.py
+++ b/pyaerocom/io/test/test_read_aeronet_sunv3.py
@@ -10,35 +10,6 @@ import os
 from pyaerocom.conftest import TEST_RTOL, testdata_unavail, lustre_unavail
 from pyaerocom.io.read_aeronet_sunv3 import ReadAeronetSunV3
 
-@lustre_unavail
-def test_load_berlin():
-    dataset = ReadAeronetSunV3()
-    files = dataset.find_in_file_list('*Berlin*')
-    assert len(files) == 1
-    assert os.path.basename(files[0]) == 'Berlin_FUB.lev30'
-    data = dataset.read_file(files[0],
-                             vars_to_retrieve=['od550aer'])
-
-    test_vars = ['od440aer',
-                 'od500aer',
-                 'od550aer',
-                 'ang4487aer']
-    assert all([x in data for x in test_vars])
-
-    # more than 100 timestamps
-    assert all([len(data[x]) > 100 for x in test_vars])
-
-    assert isinstance(data['dtime'][0], np.datetime64)
-    t0 = data['dtime'][0]
-
-    assert t0 == np.datetime64('2014-07-06T12:00:00')
-
-    first_vals = [data[var][0] for var in test_vars]
-
-    #nominal = [0.224297, 0.178662, 0.148119, 1.967039]
-    nominal = [0.22449, 0.177862, 0.147608, 1.956197]
-    npt.assert_allclose(actual=first_vals, desired=nominal, rtol=TEST_RTOL)
-
 @testdata_unavail
 @pytest.fixture(scope='module')
 def reader():

--- a/pyaerocom/io/test/test_readgridded.py
+++ b/pyaerocom/io/test/test_readgridded.py
@@ -163,7 +163,7 @@ def test_file_info(reader_reanalysis):
 
 @lustre_unavail
 def test_years_available(reader_reanalysis):
-    years = list(range(2003, 2020)) + [9999]
+    years = list(range(2003, 2021)) + [9999]
     npt.assert_array_equal(reader_reanalysis.years_avail, years)
 
 

--- a/pyaerocom/io/test/test_readungridded.py
+++ b/pyaerocom/io/test/test_readungridded.py
@@ -41,7 +41,7 @@ def test_basic_attributes():
     assert type(r.get_reader()).__name__ == 'ReadAeronetSunV3'
     assert r.dataset_provides_variables() == ['od340aer', 'od440aer',
                                               'od500aer', 'od870aer',
-                                              'ang4487aer', 'ang4487aer_calc',
+                                              'ang4487aer', 'ang44&87aer',
                                               'od550aer']
 
 @lustre_unavail

--- a/pyaerocom/test/test_ebas_sample_stats.py
+++ b/pyaerocom/test/test_ebas_sample_stats.py
@@ -12,8 +12,8 @@ import numpy as np
 from pyaerocom.conftest import lustre_unavail
 
 @lustre_unavail
-def test_ungriddeddata_jungfraujoch(data_scat_jungfraujoch):
-    data = data_scat_jungfraujoch
+def test_ungriddeddata_jungfraujoch(data_scat_jungfraujoch_full):
+    data = data_scat_jungfraujoch_full
 
     assert data.shape == (245448, 12)# (227928, 12)
     assert len(data.metadata) == 28
@@ -36,8 +36,8 @@ def test_ungriddeddata_jungfraujoch(data_scat_jungfraujoch):
                         [4.35, 7.24, 182.7, -5.57], rtol=1e-1)
 
 @lustre_unavail
-def test_scat_jungfraujoch(data_scat_jungfraujoch):
-    stat = data_scat_jungfraujoch.to_station_data('Jung*')
+def test_scat_jungfraujoch(data_scat_jungfraujoch_full):
+    stat = data_scat_jungfraujoch_full.to_station_data('Jung*')
 
     assert 'sc550aer' in stat
     assert 'sc550aer' in stat.overlap
@@ -68,10 +68,10 @@ def test_scat_jungfraujoch(data_scat_jungfraujoch):
                         [2.91, 4.58, -3.47, 97.65], rtol=1e-1)
 
 @lustre_unavail
-def test_scat_jungfraujoch_lev3(data_scat_jungfraujoch):
-    data_scat_jungfraujoch = data_scat_jungfraujoch.apply_filters(
+def test_scat_jungfraujoch_lev3(data_scat_jungfraujoch_full):
+    data_scat_jungfraujoch_full = data_scat_jungfraujoch_full.apply_filters(
             set_flags_nan=True, remove_outliers=True, data_level=2)
-    stat = data_scat_jungfraujoch.to_station_data('Jung*')
+    stat = data_scat_jungfraujoch_full.to_station_data('Jung*')
 
     assert 'sc550aer' in stat
     assert 'sc550aer' in stat.overlap
@@ -102,9 +102,9 @@ def test_scat_jungfraujoch_lev3(data_scat_jungfraujoch):
                         [2.91, 4.58, -3.47, 97.65], rtol=1e-1)
 
 @lustre_unavail
-def test_scat_jungfraujoch_subset(data_scat_jungfraujoch):
+def test_scat_jungfraujoch_subset(data_scat_jungfraujoch_full):
 
-    stat = data_scat_jungfraujoch.to_station_data('Jung*',
+    stat = data_scat_jungfraujoch_full.to_station_data('Jung*',
                                                   start=2008, stop=2011,
                                                   freq='monthly')
 

--- a/pyaerocom/test/test_obsdata_versions.py
+++ b/pyaerocom/test/test_obsdata_versions.py
@@ -26,16 +26,16 @@ def test_revision_dates():
 
     npt.assert_array_equal(res,
                            [['AeronetInvV2Lev2.daily', '20171216'],
-                            ['AeronetInvV3Lev2.daily', '20200201'],
+                            ['AeronetInvV3Lev2.daily', '20200919'],
                             ['AeronetSDAV2Lev2.daily', '20180519'],
-                            ['AeronetSDAV3Lev2.daily', '20200203'],
+                            ['AeronetSDAV3Lev2.daily', '20200922'],
                             ['AeronetSunV2Lev2.daily', '20180519'],
-                            ['AeronetSunV3Lev2.daily', '20200203'],
+                            ['AeronetSunV3Lev2.daily', '20200922'],
                             ['DMS_AMS_CVO', '20190807'],
                             ['EARLINET', '20190129'],
-                            ['EBASMC', '20191218'],
+                            ['EBASMC', '20200401'],
                             ['GAWTADsubsetAasEtAl', '20190522'],
-                            ['GHOST.daily', '20200304']])
+                            ['GHOST.EEA.daily', 'n/d']])
 
 if __name__=="__main__":
     import sys

